packages/odyssey-react/src/components/Link/Link.theme.ts

```diff
@@ -20,6 +20,7 @@ export const theme: ThemeReducer = (theme) => ({
 
   // Monochrome Variant
   MonochromeTextColor: theme.ColorTextBody,
+  MonochromeHoverTextColor: theme.ColorTextSub,
```

Comment:
Looks good to me.

Interesting we're using the subtitle color. Semantically, that doesn't make sense to me. I'd expect a muted color; something where we don't want to draw attention to it. Subtitles may or may not fit this description.

If we change the subtitle color in the future, would we also want that to be 1:1 with the hover state color? That's my question.

Reply:
Currently, we only have two a11y-friendly neutral shades for text: `TextBody` and `TextSub`. Our text sub is less "subtitle" and more "subtext" -- we tend not to use it, but when we do it's for things like hints and disclosures. Unless we redo the palette, we'll want that to cascade down to Link, Monochrome.